components/Categories.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from "react";
+import { connect } from "react-redux";
+import Category from "./Category";
+import CategoriesContainer from "./styles/CategoriesContainerStyle";
+import { fetchCategories } from "../actions";
+
+
+function Categories({ categories,fetchCategories }) {
+  useEffect(() => {
+    (async () => {
+      await fetchCategories();
```

Comment:
What is the significance of aync and await here. action(dispatch wrapper) should be passed as a props.

Reply:
done using async await, just dispatching the action